### PR TITLE
Fix CAgg on CAgg bucket size validation

### PR DESCRIPTION
**Pull request by fabriziomello**
The bucket size of a Continuous Aggregate should be greater or equal to
the parent Continuous Aggregate because there are many cases where you
actually want to roll up on another dimension.

### CODE REVIEW

File: tsl/test/expected/cagg_on_cagg_integer.out [hunk 1]
```diff
@@ -455,22 +455,19 @@ SELECT
 FROM :CAGG_NAME_1ST_LEVEL
 GROUP BY 1
 WITH NO DATA;
-psql:include/cagg_on_cagg_validations.sql:33: ERROR:  cannot create continuous aggregate with incompatible bucket width
-DETAIL:  Time bucket width of "public.conditions_summary_2" [2] should be greater than the time bucket width of "public.conditions_summary_1" [2].
```
Comment (mkindahl) [inline tsl/test/expected/cagg_on_cagg_integer.out:459]: Not part of this PR, but it might be an idea to have a look at this error message and provide some more information, e.g., the expression in each cagg that is faulty.

Comment (fabriziomello) [reply tsl/test/expected/cagg_on_cagg_integer.out:459]: I've added a DETAIL with all information regarding to the involved caggs: https://github.com/timescale/timescaledb/blob/main/tsl/src/continuous_aggs/create.c#L1350-L1361 

If you note after the cagg name you have the time bucket width inside a square brackets.

Comment (mkindahl) [reply tsl/test/expected/cagg_on_cagg_integer.out:459]: > If you note after the cagg name you have the time bucket width inside a square brackets.

Ah, didn't follow that was the width. Anyway, no need to change anything in this PR. It's already approved.

**Review by sb230132 — APPROVED** [review]
